test/proposals/Addresses.sol

```diff
@@ -508,6 +514,16 @@ contract Addresses is Test, ChainIds {
             8453,
             0x703843C3379b52F9FF486c9f5892218d2a065cC8
         );
+        _addAddress(
+            "JUMP_RATE_IRM_MOONWELL_USDC",
+            8453,
+            0x93AEE5b2431991eB96869057e98B0a7e9262cDEb
+        );
+        _addAddress(
+            "MOONWELL_USDC",
+            8453,
+            0xEdc817A28E8B93B03976FBd4a3dDBc9f7D176c22
+        );
```

Comment:
I feel that we should keep the old USDC addresses here and rename them to `DEPRECATED_` because imo it makes sense to deprecate the old market by pausing new minting with the other USDC mToken. This way, we still have those addresses in the repo and can easily interact with them in gov proposals.

Reply:
It doesn't actually make sense to deprecate the market unless the community wanted to. There is still a ton of bridged liquidity out there.

Reply:
ok, will the UI support both versions of USDC?

Reply:
it just seems that if we are going to continue supporting it or if the community votes to deprecate, in either case it would be good to have a reference to the contract in the repo so if IR model, CR, or anything else about the market changes, it can be updated without having to hunt for the address.